src/databricks/sqlalchemy/dialect/__init__.py

```diff
@@ -285,9 +285,9 @@ def has_table(
         DBR_GT_12_NOT_FOUND_STRING = "TABLE_OR_VIEW_NOT_FOUND"
 
         try:
-            res = connection.execute(
+            res = connection.execute(text(
                 f"DESCRIBE TABLE {_catalog}.{_schema}.{table_name}"
```

Comment:
what's their intent between wrapping raw queries like this?  Do they have new type-safe calls or something?

Reply:
It is more secure & resolves an inconsistency that exists between different methods of connecting to a database: 

From https://docs.sqlalchemy.org/en/20/changelog/migration_20.html 

> The removal of direct string SQL is to resolve an inconsistency between [Connection.execute()](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Connection.execute) and [Session.execute()](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.execute), where in the former case the string is passed to the driver raw, and in the latter case it is first converted to a [text()](https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.text) construct. By allowing only [text()](https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.text) this also limits the accepted parameter format to “named” and not “positional”. Finally, the string SQL use case is becoming more subject to scrutiny from a security perspective, and the [text()](https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.text) construct has come to represent an explicit boundary into the textual SQL realm where attention to untrusted user input must be given.

Reply:
gotcha.  I'm not familiar with what is going on here, so feel free to ignore.  It just seems crazy complicated if the whole point of this is to see if a table exists vs something like https://stackoverflow.com/a/64862122.  

In general what I was kind of getting at is that the use of `text()` is most likely frowned upon as a final resort and they would be moving you towards using built-in SQL functions.  Just a guess tho!